### PR TITLE
Refactoring entire program removing static, ensuring camel case, privating instance variables, and docs

### PR DESCRIPTION
**Pull request by elizabethhli**
## Linked Issue

https://github.com/autotrader-plus/autotrader-plus/issues/58
https://github.com/autotrader-plus/autotrader-plus/issues/59

## Context
Our code had used static too often, had public instance attributes, some missing documentation of methods and places where we didn't follow our naming convention of camel case.

## Your Changes
Refactored code to fix all of the above.
**Description**:

**Type of change** (select all that apply):

- [x] Refactoring (changing code structure only)

## Testing
Ran existing tests after refactoring.

## Checklist

- [x] Code is up to date with main branch (no merge conflicts)
- [x] Code has been properly tested
- [x] Code has been properly documented
- [ ] One or two people have reviewed this pull request

### CODE REVIEW

File: spring-boot/src/main/java/packages/backendlogic/Loans.java [hunk 1]
```diff
@@ -9,13 +9,12 @@
...
- * corresponding to the possible car loans for each Car based on the SensoApi
+ * Represents loans available to some given cars
  */
 public class Loans implements LoanInfoInterface{
-    private static HashMap<String, Object> loans;
-    private static CarList<Car> cars;
-    private static User buyer;
+    private final HashMap<String, Object> loans;
+    private final CarList<Car> cars;
+    private final User buyer;
```
Comment (parthaba) [inline spring-boot/src/main/java/packages/backendlogic/Loans.java:17]: don't make variables final

File: spring-boot/src/main/java/packages/informationmanipulation/ReturnMultipleCars.java [hunk 2]
```diff
@@ -9,28 +9,33 @@
...
+    private ArrayList<HashMap<String, Object>> returnAllCarsQuery(String query) throws SQLException{
 
         // Establish a connection and create a set of results from that query
         AutoTraderDBInterface connection = new ConnectAutoTraderDB();
         ResultSet myResultSet = connection.writeQuery(query);
 
         ArrayList<HashMap<String, Object>> returnList = new ArrayList<>();
 
+        // Create a ReturnCarInformation Object to be used in the while loop
+        ReturnCarInformation r = new ReturnCarInformation();
```
Comment (parthaba) [inline spring-boot/src/main/java/packages/informationmanipulation/ReturnMultipleCars.java:32]: Maybe look into making more descriptive variable names, r doesn't explain much

File: spring-boot/src/main/java/packages/returninfo/ServerMainEndpointHandler.java [hunk 3]
```diff
@@ -99,7 +100,8 @@ public String httpResponseSenso(@RequestBody() String req_body) {
      */
     private ArrayList<HashMap<String, Object>> getAllCars() throws DatabaseConnectionFailureException {
         try {
-            return ReturnMultipleCars.returnAllCars();
+            ReturnMultipleCars r = new ReturnMultipleCars();
```
Comment (parthaba) [inline spring-boot/src/main/java/packages/returninfo/ServerMainEndpointHandler.java:103]: Change variable name

File: spring-boot/src/test/java/packages/backendlogic/LoansTest.java [hunk 4]
```diff
@@ -17,7 +17,8 @@
 
     @BeforeEach
     void setUp() throws SQLException, SensoConnectionFailureException {
-        carlist = ReturnMultipleCars.returnAllCars();
+        ReturnMultipleCars r = new ReturnMultipleCars();
```
Comment (parthaba) [inline spring-boot/src/test/java/packages/backendlogic/LoansTest.java:20]: Change variables

File: spring-boot/src/test/java/packages/informationmanipulation/AddUserTest.java [hunk 5]
```diff
@@ -8,13 +9,20 @@
 import java.util.HashMap;
 
 class AddUserTest {
+    ReturnUserInformation r;
```
Comment (parthaba) [inline spring-boot/src/test/java/packages/informationmanipulation/AddUserTest.java:12]: Change variable

File: spring-boot/src/test/java/packages/informationmanipulation/AddUserTest.java [hunk 6]
```diff
@@ -8,13 +9,20 @@
...
+        r = new ReturnUserInformation();
+    }
 
     @Test
     @DisplayName("addUser: Basic Case with Simple User")
     void addUserSimple() throws SQLException {
         User user = new User(740, 1000, 5000, "M4Y111", "Mike");
 
-        AddUser.addUser(user);
+        AddUser a = new AddUser();
```
Comment (parthaba) [inline spring-boot/src/test/java/packages/informationmanipulation/AddUserTest.java:24]: You really don't like using real words huh

**Review by parthaba — CHANGES_REQUESTED** [review]

File: spring-boot/src/test/java/packages/informationmanipulation/AddUserTest.java (hunk 5 above)
Comment (parthaba) [reply spring-boot/src/test/java/packages/informationmanipulation/AddUserTest.java:12]: There are a bunch more occasions where this happens, not sure if you want to change but I'd recommend doing so.

File: spring-boot/src/main/java/packages/connectouterentity/ConnectSensoAPI.java [hunk 7]
```diff
@@ -15,17 +15,17 @@
 /** This is a class that calls Senso API, and return information from Senso API.**/
```
Comment (parthaba) [inline spring-boot/src/main/java/packages/connectouterentity/ConnectSensoAPI.java:15]: Resolve conflicts before merging

File: spring-boot/src/main/java/packages/backendlogic/Loans.java [hunk 8]
```diff
@@ -9,13 +9,12 @@
 import java.util.HashMap;
```
Comment (parthaba) [inline spring-boot/src/main/java/packages/backendlogic/Loans.java:9]: Resolve conflicts before merging

**Review by parthaba — CHANGES_REQUESTED** [review]
Basic conflict resolution

File: spring-boot/src/main/java/packages/informationmanipulation/ReturnMultipleCars.java (hunk 2 above)
Comment (jiah29) [reply spring-boot/src/main/java/packages/informationmanipulation/ReturnMultipleCars.java:32]: hahaha r

**Review by Danx1129 — APPROVED** [review]
Lots of edits, but everything seems to be good

**Review by jiah29 — APPROVED** [review]

**Review by jiah29 — APPROVED** [review]

**Pull request merged**